wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/integration/BroadcastIntegrationTest.java

```diff
@@ -0,0 +1,296 @@
+        Assertions.assertThat(message.getNachricht()).isEqualTo("Das ist ein Test");
+    }
+
+    @Test
+    public void getMessageIntegrationTestGetParamBlank() {
+        log.debug("#GetMessageIntegrationTestGetParamBlan");
+        Exception catchedException1 = null;
+        try {
+            mvc.perform(
+                    get(GETMESSAGE_URL + "    ")
+                            .contentType(MediaType.APPLICATION_JSON_UTF8)
+                            .accept(MediaType.APPLICATION_JSON))
+                    .andReturn().getResponse();
+
+        } catch (Exception e) {
+            catchedException1 = e;
+        }
+
+        Assertions.assertThat(catchedException1)
+                .isNotNull()
+                .isInstanceOf(ServletException.class)
+                .hasMessageContaining("wahlbezirkID is blank or empty");
+    }
+
+    @Test
+    public void getMessageIntegrationTestGetParamEmpty() throws Exception {
+        log.debug("#GetMessageIntegrationTestGetParamEmpty");
+        String wahlbezirkID = "";
+        mvc.perform(get(GETMESSAGE_URL + wahlbezirkID)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound())
+                .andExpect(result -> {
+                    Map<String, Object> mmodel = result.getModelAndView().getModel();
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException")
+                            .isInstanceOf(TechnischeWlsException.class)
+                            .extracting("code", "serviceName")
+                            .contains("999", serviceOid);
+
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException.cause")
+                            .isInstanceOf(NoResourceFoundException.class);
+                });
+    }
+
+    @Test
+    public void getMessageNoContentIntegrationTest() {
+        log.debug("#GetMessageNoContentIntegrationTest");
+
+        ServletException thrownException = null;
+        try {
+            mvc.perform(
+                    get(GETMESSAGE_URL + "123")
+                            .contentType(MediaType.APPLICATION_JSON_UTF8)
+                            .accept(MediaType.APPLICATION_JSON));
+        } catch (Exception e) {
+            thrownException = (ServletException) e;
+        }
+
+        Assertions.assertThat(thrownException.getCause())
+                .isNotNull()
+                .isInstanceOf(FachlicheWlsException.class)
+                .hasMessageContaining("No message found")
+                .extracting("code", "serviceName")
+                .contains("204", serviceOid);
+    }
+
+    @Test
+    public void deleteIntegrationTest() throws Exception {
```

Comment:
Schreibt man mit Junit 5 nicht mehr

```suggestion
    void deleteIntegrationTest() throws Exception {
```